src/hotspot/share/interpreter/linkResolver.cpp

```diff
@@ -1150,7 +1150,7 @@ Method* LinkResolver::linktime_resolve_special_method(const LinkInfo& link_info,
   // and the selected method is recalculated relative to the direct superclass
   // superinterface.method, which explicitly does not check shadowing
   Klass* resolved_klass = link_info.resolved_klass();
-  Method* resolved_method;
+  Method* resolved_method = NULL;
```

Comment:
Why would it complain about this one?  resolved_method is going to get some value in the next 4 lines.

Reply:
Ah yes, it does not complain about this one. I just initialized all locals that were introduced in your original change.

Reply:
I don't think it matters that much.  Go ahead and leave it.  I'm glad it didn't complain about that.